.github/workflows/run-benchmark.yml

```diff
@@ -0,0 +1,163 @@
+name: benchmark
+on:
+  workflow_call:
+
+jobs:
+  benchmark-vs-thresholds:
+    # https://runs-on.com/features/custom-runners/
+    runs-on:
+      labels:
+        - runs-on
+        - runner=2cpu-4ram
+        - run-id=${{ github.run_id }}
+
+    container: swift:noble
+
+    defaults:
+      run:
+        shell: bash --noprofile --norc -e -u -o pipefail {0}
+
+    env:
+      PR_COMMENT: null # will be populated later
+
+    steps:
+      - name: Configure RunsOn
+        uses: runs-on/action@v1
+
+      - name: Check out code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Configure git
+        run: git config --global --add safe.directory "${GITHUB_WORKSPACE}"
+
+      # jemalloc is a dependency of the Benchmarking package
+      # actions/cache will detect zstd and will become much faster.
+      - name: Install jemalloc, curl, jq and zstd
+        run: |
+          apt-get update -y
+          apt-get install -y libjemalloc-dev curl jq zstd
+
+      - name: Restore .build
+        id: restore-cache
+        uses: actions/cache/restore@v4
+        with:
+          path: Benchmarks/.build
+          key: "swiftpm-benchmark-build-${{ runner.os }}-${{ github.event.pull_request.base.sha || github.event.after }}"
+          restore-keys: "swiftpm-benchmark-build-${{ runner.os }}-"
+
+      - name: Run benchmarks for branch '${{ github.head_ref || github.ref_name }}'
+        run: |
+          swift package -c release --disable-sandbox \
+            --package-path Benchmarks \
+            benchmark baseline update \
+            '${{ github.head_ref || github.ref_name }}'
+
+      - name: Read benchmark result
+        id: read-benchmark
+        run: |
+          swift package -c release --disable-sandbox \
+            --package-path Benchmarks \
+            benchmark baseline read \
+            '${{ github.head_ref || github.ref_name }}' \
+            --no-progress \
+            --format markdown \
+            >> result.text
```

Comment:
```suggestion
            > result.text
```

Reply:
that would be fine but i thought i keep everything with >> for consistency, otherwise did notice this when writing the original file. Still fine by me either way.